### DISCUSSION TITLE
New Question types

### DISCUSSION THREAD

**Discussion by jcano897** [Arquisoft/wiq_en1b#91] [Ideas]
Leave here any idea for possible question types to be implemented. The idea is to have at least 10 different types. Going over 10 would be more a problem of the frontend (how to represent it in the options) more than a problem in the generator.

Current types:
* Capital
* Population
* Size
* Language
* Head of Government
* Film director
* Videogame developer
* Videogame publisher
* Videogame genre
* Videogame country

Already reviewed types:
* Accepted (not yet introduced):
* Rejected
    * Head of state: due to the similarity with head of government this was not included, but if the team deems it so it could potentially replace head of government or be included alongside it.
    * Author of a book: This is becasue the author property is extremelly prevalent within wikidata, but not directly, so if we ask to give us entites that have that property, the chance of the entites actually having said property are less than 10%, because the entites will instead link to something (that may link to something, that may link to something, and so on), that will have the property.
* Under investigation
    * Performer of a song

**Comment by ErdemYabaci** [comment]
Performer of a song.
Writer of a book.
Director of a film.
